webknossos/webknossos/geometry/bounding_box.py

```diff
@@ -31,7 +33,7 @@ class BoundingBox:
     topleft: Vec3Int = attr.field(converter=Vec3Int)
     size: Vec3Int = attr.field(converter=Vec3Int)
     bottomright: Vec3Int = attr.field(init=False)
-    name: Optional[str] = "Unnamed Bounding Box"
+    name: Optional[str] = _DEFAULT_BBOX_NAME
     is_visible: bool = True
     id: Optional[str] = None
```

Comment:
Maybe the id should rather be an int?

Reply:
It’s int32 in the proto definition so int sounds right to me